### PR TITLE
MANIFEST.in: add CHANGELOG.md and tests/ to sdist

### PR DESCRIPTION
**Pull request by anthrotype**
Without this patch, installing from the unicodedata2.tar.gz source distribution package produces an error as setup.py script attempts to read from the 'CHANGELOG.md' file to generate the long_description, but does not find it in the tarball.

### CODE REVIEW

**Comment by anthrotype** [comment]
@mikekap can you please delete the broken tarball on PyPI for the 13.0.0 release https://files.pythonhosted.org/packages/9c/30/b335ab574335f7f6ca51ad1247a59d331637b59699a26cdf0c9b5832861a/unicodedata2-13.0.0.tar.gz ?

After we merge this PR, we could try to push a new git tag and Travis should upload the fixed tar.gz file.
Thanks

**Comment by anthrotype** [comment]
actually, we can't do that because of https://github.com/pypa/packaging-problems/issues/74

If we want to fix the broken sdist, we will have to bump the setup.py version (which we wouldn't like to do, since unicodedata2 versions follow Unicode Data's own version).

The only solution I see is to do a post-release https://www.python.org/dev/peps/pep-0440/#post-releases and add a suffix like `.post1` to the version string.

I know it's not ideal, and confuses some people.
The other option is don't fix it and wait for next unicode release..

We do provide most, if not all the binary wheels, so it is unlikely that one hits this problem.
Basically this only ever occurs when none of the available wheels matches the user's local platform/python version.

Sorry, I added that thing that reads from CHANGELOG.md in setup.py wirh #32, but forgot to check that sdist was still working after that...

**Comment by mikekap** [comment]
@anthrotype A post version makes sense to me. It's not unprecedented in this package either - see https://github.com/mikekap/unicodedata2/blob/master/CHANGELOG.md#900-4 . On a related note - it could be worth revisiting the versioning scheme given we do sometimes make slight changes.

I think you do quite a bit of maintaining this package so don't worry about the bug - it would be surprising if we never shipped bugs. That said, in the spirit of "hope is not a strategy" we can add some tests around installing the artifacts (sdists & wheels). It's debatable how many other issues that can catch though. What do you think?

**Comment by anthrotype** [comment]
Ok, then I propose we use a `X.Y.Z-N` version scheme, with X.Y.Z matching Unicode version, and incremental `N` (starting from 1) post-releases to fix up bugs in previous releases.
I'll cut a 13.0.0-1 shortly after merging this PR.

> we can add some tests around installing the artifacts (sdists & wheels).

If we use `tox` as our test runner, we would catch this issues, since tox creates a new test venv, builds a sdist archive, then installs that sdist into the venv and runs the test suite, all automatically.

I'll send a PR for that when I get the chance (possibly after the 13.0.0-1 release).
